_source/providenceConfiguration/developer/media_volumes_description.csv

```diff
@@ -0,0 +1,9 @@
+Key,Description,Example
+hostname,Hostname,<site_hostname>
+protocol,"Protocol (ftp, http[s], gopher, etc.)",<site_protocol>
+urlPath,"Base path for exposing the media on this volume",<ca_media_url_root>/images
+absolutePath,"Absolute filesystem path for the volume",<ca_media_root_dir>/images
+writeable,"A flag to indicate if the volume is writeable or not",1
+description,"Description of the volume",Images
+accessUsingMirror,"TBD",
```

Comment:
Definition of this option:

Name of mirror to serve media from, when available. If the specified mirror does not yet have the required media, the local copy will be served.